server/src/main/java/org/candlepin/model/PoolCurator.java

```diff
@@ -2006,4 +2026,93 @@ public void clearPoolSourceEntitlementRefs(Iterable<String> poolIds) {
 
         return output;
     }
+
+    /**
+     * Fetches a collection of active pool IDs mapped to a given owner. If no such pool
+     * are present, an empty collection will be returned.
+     *
+     * @param ownerId
+     *  The ID of the owner for which to fetch active Pool IDs
+     *
+     * @return
+     *  A collection of active Pool IDs belonging to the given owner.
+     */
+    public Collection<String> getActivePoolByOwnerId(String ownerId) {
+        Date currentDate = new Date();
+        String jpql = "SELECT p.id FROM Pool p WHERE p.owner.id = " +
```

Comment:
Try not to put line breaks for continuations in the middle of a statement or clause. There's enough space for the :owner_id part of the first WHERE clause, or the WHERE portion should move down to the line with it.

Reply:
Noted